packages/components/src/components/dropdown/_dropdown.scss

```
@@ -327,11 +334,11 @@
       outline: none;
     }
 
-    .#{$prefix}--dropdown-text {
+    .#{$prefix}--list-box__label {
```

```suggestion
    // TODO: remove in v11
    .#{$prefix}--dropdown-text,
    .#{$prefix}--list-box__label {
```

Might help to keep the previous selector around until next release to protect against breaking changes